erpnext/patches/v15_0/migrate_payment_request_status.py

```diff
@@ -0,0 +1,15 @@
+import frappe
+
+
+def execute():
+	"""
+	Description:
+	Change Inward Payment Requests from statut 'Initiated' to correct status 'Requested'.
+	Status 'Initiated' is reserved for Outward Payment Requests and was a semantic error in previour versions.
+	"""
+
+	if frappe.reload_doc("accounts", "doctype", "Payment Request"):
```

Comment:
This patch will never execute because it's in post section where it's already reloaded.

Reply:
Oh damn, sry! Is this the right fix? https://github.com/frappe/erpnext/pull/38045